sapi/litespeed/lsapi_main.c

```diff
@@ -865,7 +865,7 @@ static int lsapi_activate_user_ini_finally(_lsapi_activate_user_ini_ctx *ctx,
     return rc;
 }
 
-static int lsapi_activate_user_ini(TSRMLS_D)
+static int lsapi_activate_user_ini( void )
```

Comment:
@krakjoe the sapi/litespeed code is afair the same in all branches. Think that's what @nikic means, so it's easier to maintain. Much like mysqlnd wad in 5.x